tests/bin/install-wc-tests.sh

```diff
@@ -10,20 +10,28 @@ DB_USER=$2
 DB_PASS=$3
 DB_HOST=${4-localhost}
 WP_VERSION=${5-latest}
-WC_VERSION=${6-"5.0.0"}
+WC_VERSION=${6-"7.4.1"}
 
-install_wc() {
-	git clone --depth=1 --branch=$WC_VERSION https://github.com/woocommerce/woocommerce.git /tmp/woocommerce
-	composer install -d /tmp/woocommerce
+export NVM_DIR="$HOME/.nvm"
+NVM_PATH=/opt/homebrew/opt/nvm/nvm.sh
+
+# Load nvm.
+. $NVM_PATH
 
-	echo "Done unzipping WooCommerce\n"
+install_wc() {
+	git clone --depth=1 --branch=$WC_VERSION https://github.com/woocommerce/woocommerce.git /tmp/woocommerce && cd /tmp/woocommerce/plugins/woocommerce && nvm use && composer install && pnpm run build:feature-config && echo "Done unzipping WooCommerce"
```

Comment:
Instead of `nvm use`, what about `nvm i` instead? The system might not have the version installed and it will throw an error. `nvm i` is like a superset of `use` in the event the version isn't installed.

Reply:
Love it!